skynet/utils.py

```diff
@@ -46,3 +47,11 @@ async def create_webserver(app, port):
     )
     server = uvicorn.Server(server_config)
     await server.serve()
+
+
+def get_device() -> str:
```

Comment:
These logs are going to be emitted every time the function is called. I'd cache the result.

Reply:
This is just a function I've moved from the streaming whisper module, from what I'm seeing it is only used at startup

Reply:
Ack!